code/game/mecha/equipment/tools/work_tools.dm

```diff
@@ -455,6 +455,43 @@
 	last_piece = NC
 	return TRUE
 
+/obj/item/mecha_parts/mecha_equipment/mech_crusher
+	name = "exosuit crusher"
+	desc = "A mech mounted crusher. For crushing bigger things."
+	icon_state = "mecha_crusher"
+	equip_cooldown = 15
+	energy_drain = 3000
+	harmful = TRUE
+	range = MECHA_MELEE | MECHA_RANGED
+	var/obj/item/kinetic_crusher/mecha/internal_crusher = new
```

Comment:
if you instantiate this in nullspace, when the mech crusher gets destroyed, it can't be GCed. put it in Initialize, make it part of the mech crusher's contents, and qdel it on destroy